mesonbuild/msubprojects.py

```diff
@@ -435,6 +436,33 @@ def update(self) -> bool:
             self.wrap.update_hash_cache(self.repo_dir)
         return success
 
+    def syncwrap(self) -> bool:
+        if self.wrap.type != 'git' and not self.wrap.redirected:
```

Comment:
Also should skip if `not os.path.isdir(self.repo_dir)`.

Reply:
will do.